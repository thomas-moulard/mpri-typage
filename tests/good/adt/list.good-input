type list =
  Nil 
| Cons of int * list
end

val l = Cons 0 Nil

