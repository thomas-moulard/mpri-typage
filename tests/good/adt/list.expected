Nil = Nil

Cons = Cons

l = Cons 0 Nil

l : list
Cons : (int -> (list -> list))
Nil : list
